tfrecord/torch/dataset.py

```diff
@@ -154,7 +155,7 @@ def __init__(self,
         super(MultiTFRecordDataset, self).__init__()
         self.data_pattern = data_pattern
         self.index_pattern = index_pattern
-        self.splits = splits
+        self.splits = sorted(list(splits.items()))
```

Comment:
What's the purpose of this?

Reply:
The reasoning for this stems from the reasoning explained by comment below. Of course the library does not guarantee ordered reading of the data, but you need to have stable ordering in order to give appropriate part of the files to the corresponding worker.

I thought it makes sense to do this conversion to sliceable list, that is ordered across all workers only once in the `__init__` instead of doing it for each worker in `__iter__`